src/main/java/uk/gov/hmcts/reform/userprofileapi/config/RestAuthenticationEntryPoint.java

```diff
@@ -29,7 +29,7 @@ public void commence(HttpServletRequest request, HttpServletResponse response,
         response.setStatus(HttpServletResponse.SC_UNAUTHORIZED);
         String errorMessage = mapper.writeValueAsString(errorResponse);
         response.setHeader("UnAuthorized-Token-Error", errorMessage);
-        log.error(errorMessage);
+        log.error("RD TEAM" + errorMessage);
```

Comment:
this log will be printed in prod as well. Can you add separate log.debug for this?